src/fswAlgorithms/attDetermination/sunlineEphem/_UnitTest/test_sunlineEphem.py

```diff
@@ -160,22 +158,15 @@ def sunlineEphemTestFunction(show_plots):
                 [0.0, 0.0, 0.0],
                 ]
 
-    # compare the module results to the truth values
-    accuracy = 1e-12
-    for i in range(0,len(trueVector)):
-        # check a vector values
-        if not unitTestSupport.isArrayEqual(estVector[i], trueVector[i], 3, accuracy):
-            testFailCount += 1
-            testMessages.append("FAILED: " + sunlineEphemObj.modelTag + " Module failed sunlineEphem " +
-                                " unit test at t=" + str(dataLog.times()[i]*macros.NANO2SEC) + "sec\n")
-
-    # print out success message if no error were found
-    assert testFailCount == 0, testMessages
-    print(f"PASSED: {sunlineEphemObj.modelTag}")
+    # one assert; on failure numpy.testing will show exactly where the mismatch is
+    for i in range(len(trueVector)):
+        np.testing.assert_almost_equal(estVector[i], trueVector[i],
+                                       decimal=12,
+                                       err_msg=f"FAILED: {sunlineEphemObj.modelTag}. Unit test failed at {i}-th testVectors",
```

Comment:
:small_red_triangle_down: You can remove the FAILED from here as pytest conveys that information.